src/pages/Haha/TodoList/AddItems.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import Input from './components/Input';
+import Button from './components/Button';
+
+function AddItems({ add, handleDialog }) {
+  const [text, setText] = useState({ val: '', isDone: false, id: Date.now() });
+
+  const onChange = ({ target }) => {
+    setText({ ...text, val: target.value });
+  };
+
+  const onKeyDown = ({ key }) => {
+    if (key === 'Enter') return onClick();
```

Comment:
實際上 `onClick` 是一個操作，他不具有回傳值，不用 `return`
可採用更符合的if用法

```suggestion
    key === 'Enter' && onClick();
```